instrumentation/packaging/fpm/rpm/build.sh

```diff
@@ -27,8 +27,8 @@ if [[ -z "$VERSION" ]]; then
     VERSION="$( get_version )"
 fi
 
-# rpm doesn't like dashes in the version, replace with tildas
-VERSION="${VERSION/'-'/'~'}"
+# rpm doesn't like dashes in the version, replace with underscore
+VERSION="${VERSION/'-'/'_'}"
```

Comment:
Is there any chance someone's relying on the version to be a `~` here? I'm wondering if we're going to break anyone's automation.

This probably isn't a big deal either way, as long as it's called out in the release notes.

Reply:
This change only affects packages built from `main` for dev/test. It does not affect any packages for actual releases/tags.